### PR TITLE
Add history plugin

### PR DESCRIPTION
**Pull request by wd**
This plugin could help people list and diff versions of deployment/daemonset/statefulset. 

The detailed explanation could be found at here: https://github.com/wd/kubectl-history

### CODE REVIEW

**Comment by ahmetb** [comment]
:robot: _Beep beep! I’m a robot speaking on behalf of @ahmetb._ :robot:

-----

Thanks for submitting your kubectl plugin to Krew!
One of the krew-index maintainers will review it soon. Note that the reviews for new plugin submissions may take a few days.

In the meanwhile, here are a few tips to make your plugin manifest better:
* Make sure your plugin follows the [best practices](https://krew.sigs.k8s.io/docs/developer-guide/develop/best-practices/).
* Eliminate redundant wording form `shortDescription` (it should be max 50 characters).
* Try to word wrap your `description` to 80-character lines (no usage examples, please).

Thanks for your patience!
/kind new-plugin

**Comment by wd** [comment]
hi @ahmetb, could give me some guidance for how to proceed this PR? Thank you!

File: plugins/history.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: history
+spec:
+  version: v1.1.8
+  homepage: https://github.com/wd/kubectl-history
+  shortDescription: Plugin to List and diff history reversions of deployment/daemonset/statefulset
```
Comment (ahmetb) [inline plugins/history.yaml:8]:
```suggestion
  shortDescription: List & diff revisions of workload resources
```

please keep this short ~50 chars as it's truncated.

**Comment by ahmetb** [comment]
Sorry it took a while. I did a quick review. Can merge now or later, your call.

Also, please consider setting up [Krew release automation](https://krew.sigs.k8s.io/docs/developer-guide/release/automating-updates/) which helps you skip manually making updates to your Krew manifests on each new version and send a pull request. It’s a GitHub workflow bot that can run every time you push a new tag. Those PRs are automatically approved.

**Comment by wd** [comment]
thank you! I already have added the bot to the workflow https://github.com/wd/kubectl-history/blob/main/.github/workflows/release.yaml#L34

It looks like I don't have the merge permission. It would be great if you help me merge it. Thanks.

**Comment by ahmetb** [comment]
/lgtm
/approve
Thanks!

**Pull request merged**